src/custom/simplifions/components/SimplifionsSolutionDescription.vue

```diff
@@ -137,38 +137,50 @@
     </figure>
 
     <div class="fr-col-12 fr-col-md-8 fr-mb-4w">
-      <h2 id="possibilites-simplification" class="colored-title fr-h2 fr-my-5w">
-        Possibilités de simplification :
-      </h2>
-      <!-- eslint-disable-next-line vue/no-v-html -->
-      <p v-html="fromMarkdown(solution.Description_longue)"></p>
-      <p>
-        <strong>
-          <span
-            aria-hidden="true"
-            style="color: #27a658"
-            class="fr-icon-success-fill"
-          ></span>
-          Cette solution permet :
-        </strong>
-      </p>
+      <div v-if="solution.Description_longue">
+        <h2
+          id="possibilites-simplification"
+          class="colored-title fr-h2 fr-my-5w"
+        >
+          Possibilités de simplification :
+        </h2>
+        <!-- eslint-disable-next-line vue/no-v-html -->
+        <p v-html="fromMarkdown(solution.Description_longue)"></p>
+      </div>
+
+      <div v-if="solution.Cette_solution_permet_">
+        <p>
+          <strong>
+            <span
+              aria-hidden="true"
+              style="color: #27a658"
```

Comment:
Should `style` be extracted? Really just a question since you're in "custom".

Reply:
Done it